src/Impostor.Api/Innersloth/Language.cs

```diff
@@ -0,0 +1,11 @@
+namespace Impostor.Api.Innersloth
+{
+    public enum Language
```

Comment:
Are these the same languages that are used for searching for games an such?

Reply:
Nope, you are talking about [GameKeywords](https://github.com/Impostor/Impostor/blob/dev/src/Impostor.Api/Innersloth/GameKeywords.cs), difference is that GameKeywords is only for room chat language while Language is whole among us translation (also different values for enums)
![image](https://user-images.githubusercontent.com/35262707/102354156-767fa880-3faa-11eb-9b76-c8512ffef84a.png)